ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-presubmits.yaml

```diff
@@ -8060,6 +8060,90 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )(azure-4.17-nightly-x86-data-path-9nodes|remaining-required),?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build09
+    context: ci/prow/azure-4.17-nightly-x86-data-path-ipsec-9nodes
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+      timeout: 7h0m0s
+    labels:
+      ci-operator.openshift.io/cloud: azure4
+      ci-operator.openshift.io/cloud-cluster-profile: azure-qe
+      ci-operator.openshift.io/variant: azure-4.17-nightly-x86
+      ci.openshift.io/generator: prowgen
+      job-release: "4.17"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-data-path-ipsec-9nodes
+    rerun_command: /test azure-4.17-nightly-x86-data-path-ipsec-9nodes
+    spec:
```

Comment:
@qiliRedHat Can you add Slack Notifications as mentioned here:
https://github.com/openshift/release/pull/57642/files
```
  reporter_config:
    slack:
      channel: '#ocp-qe-scale-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :white_check_mark: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :white_check_mark:
        {{else}} :warning:  Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> :warning: {{end}}'
```

Reply:
@krishvoor Do we need to add the slack notificationto a presubmits job?

Reply:
I see the change in job name here:
https://github.com/openshift/release/pull/56033/files#diff-0deaef2ed03ee43e08edd99238304c2f6692920978ab6fd305fcfa7f4a9b9bcdL54 
Lgtm